### DISCUSSION TITLE
Syslog filled with casaos entries

### DISCUSSION THREAD

**Discussion by suncobran** [IceWhaleTech/CasaOS#1448] [Q&A]
I have an Ubuntu 22.04.3 LTS machine with CasaOs installed. After several weeks of usage i had to restart it an found out during booting, that the disk is full, and that the OS cant start.  The **syslog** was 400GB+ large. It was filled with the following 5 casaos entries:

`
Oct 15 17:59:32 MyPC casaos-app-management[3937]: {"time":"2023-10-15T17:59:32.531872818+02:00","id":"","remote_ip":"127.0.0.1","host":"localhost","method":"GET","uri":"/v2/app_management/web/appgrid","user_agent":"Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/118.0","status":200,"error":"","latency":31722878,"latency_human":"31.722878ms","bytes_in":0,"bytes_out":2085}

Oct 15 17:59:32 MyPC casaos-user-service[1729]: [GIN] 2023/10/15 - 17:59:32 | 200 |     344.346µs |       127.0.0.1 | GET      "/v1/users/current/custom/link"

Oct 15 17:59:32 MyPC casaos-user-service[1729]: [GIN] 2023/10/15 - 17:59:32 | 200 |     332.892µs |       127.0.0.1 | GET      "/v1/users/current/custom/app_order"

Oct 15 17:59:33 MyPC casaos[1582]: /bin/bash -c source /usr/share/casaos/shell/helper.sh ;CatNetCardState enp5s0

Oct 15 17:59:33 MyPC casaos-message-bus[1417]: {"time":"2023-10-15T17:59:33.00691215+02:00","id":"","remote_ip":"127.0.0.1","host":"127.0.0.1:42691","method":"POST","uri":"/v2/message_bus/event/casaos/casaos:system:utilization","user_agent":"Go-http-client/1.1","status":200,"error":"","latency":261992,"latency_human":"261.992µs","bytes_in":649,"bytes_out":806}
`
Anybody with this issue?

**Comment by SpockIsCoding** [comment]
did you solve that problem ?

**Reply by suncobran** [reply]
No. I try to regularly empty the syslog to avoid the problem. Sorry for late reply.

**Reply by welovewebs** [reply]
Same here, tried Jammy and Bullseye both in Incus (LXD) containers using ZFS, on Bookworm host.
Constantly logging to syslog with messages very similar to suncobran

**Comment by strouja** [comment]
@suncobran @welovewebs I noticed the same issue today.  There are 2 issues and both will cause your fileystem to fill up and make your system not usable.   See my comment [here](https://github.com/IceWhaleTech/CasaOS/issues/1336#issuecomment-2509398221) related to this and fixing incorrect timestamps on files which prevented me from having the log rotation work.   I hyperlinked that issue with this [comment](https://github.com/IceWhaleTech/CasaOS/issues/1472#issuecomment-2509387995)  on how to set up log rotation so the filesytem will not get filled up.

But I have no idea how to fix the root cause with `/usr/share/casaos/shell/helper.sh` which is causing my `/var/log/syslog` file to grow excessively

**Comment by greenaar** [comment]
I added this to /etc/rsyslog/05-casaos.conf

```
if $msg contains 'casa' then {
    stop
}

if ( $programname == "casaos" ) then {
  stop
}
```

This filters those messages out, as I don't actually care about them in my environment, otherwise I would've added

action(type="omfile" file="/var/log/casaos.log")

before the 'stop'
